strutil/strutil.go

```diff
@@ -365,3 +365,12 @@ func WordWrapPadded(out io.Writer, text []rune, pad string, termWidth int) error
 	}
 	return WordWrap(out, text, indent, indent, termWidth)
 }
+
+// NumItems returns the string form of the number of items: the singular form
+// (eg: "1 item") if n is 1, otherwise the plural form (eg: "7 items").
+func NumItems(n int, singular, plural string) string {
```

Comment:
You could take some inspiration from the i18n signature if you want.

<snapd>/i18n/i18n.go:

// G is the shorthand for Gettext
func G(msgid string) string { ... }

// NG is the shorthand for NGettext
func NG(msgid string, msgidPlural string, n int) string { ... }